exercises/poker/src/example/java/Poker.java

```diff
@@ -0,0 +1,142 @@
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class Poker {
+	private static class Card {
+		private int rank;
+		private int suit;
+
+		public Card(String card) {
+			rank = parseRank(card);
+			suit = parseSuit(card);
+		}
+
+		private int getRank() {
+			return rank;
+		}
+
+		private int getSuit() {
+			return suit;
+		}
+
+		private static int parseRank(String card) {
+			return "..23456789TJQKA".indexOf(card.charAt(0));
+		}
+
+		private static int parseSuit(String card) {
+			return ".HSDC".indexOf(card.charAt(1));
+		}
+	}
+
+		private static class Hand {
+			private String input;
+			private int score;
+
+			private Hand(String hand) {
+				this.input = hand;
+				this.score = scoreHand(parseCards(hand));
+			}
+
+			private int getScore() {
+				return score;
+			}
+
+			private String getInput() {
+				return input;
+			}
+
+			private static List<Card> parseCards(String hand) {
+				ArrayList<Card> parseCards = new ArrayList<Card>();
```

Comment:
To avoid confusion with the method name, maybe name this variable `parsedCards` instead?